moja-kuhinjica-frontend/src/pages/index.tsx

```diff
@@ -22,8 +23,7 @@ const Home = () => {
 
     const [showLoginModal, setShowLoginModal] = useState(false)
     const [showSignUpModal, setShowSignUpModal] = useState(false)
-    const [showSignUpModalNotification, setShowSignUpModalNotification] =
-        useState(false)
+    const [showNotification, setNotification] = useState(false)
```

Comment:
You made typo here. If state is `showNotification`, then state update function should be `setShowNotification` instead of `setNotification`